torch/_inductor/mkldnn_utils.py

```diff
@@ -0,0 +1,28 @@
+import torch
+
+# check the pattern: (nn.module, F.function) matched.
+def matches_module_function_pattern(pattern, node, modules):
```

Comment:
It is a general function, not mkldnn specific? Why do we move it to mkldnn_util.py?

Reply:
Yes, how about fx_utils.py?